app/src/main/java/edu/uwstout/p2pchat/ChatFragment.java

```diff
@@ -81,10 +81,10 @@ public void onClick(final View view)
 
                 binding.textInput.setText("");
 
-                //An inmemory file.
+                //A new InMemoryFile created with the text.
```

Comment:
I still don't think this comment adds any value to the code but that might just be me being nit-picky. If you want to keep this comment I'll let it pass.